symplyphysics/laws/dynamics/vector/torque_vector_of_twisting_force.py

```diff
@@ -0,0 +1,30 @@
+from symplyphysics import (
+    units,
+    QuantityVector,
+    Vector,
+    cross_cartesian_vectors,
+    validate_input,
+    validate_output,
+    list_of_quantities,
+)
+
+# Description
+## Torque is a turning or twisting action on a body about a rotation axis due to a force.
+## It is defined as a cross product of the force vector and the position vector of the point
+## of force application.
```

Comment:
You can add somewhere a note, that torque is a pseudovector: https://en.wikipedia.org/wiki/Pseudovector